spec/koans.js

```diff
@@ -8,31 +8,29 @@ describe('`let` restricts the scope of the variable to the current block - ', ()
   describe('`let` vs. `var`.', () => {
     it('`var` works as usual, it does not restricts scope', () => {
       if (true) {
-        /*You should add your code in here*/
+       var varX = true;/*You should add your code in here*/
       }
-      // expect(varX).toBe(true);
+       expect(varX).toBe(true);
     });
 
     it('`let` restricts scope to inside the block', () => {
-      /*var or const? letX = false*/
+      const letX = false /*var or const? letX = false*/
       if (true) {
-        /*var or const? letX = true*/
+       const letX = true /*var or const? letX = true*/
       }
-      //expect(letX).toBe(false);
+      expect(letX).toBe(false);
     });
 
     it('`var` does not restricts scope to inside the block in `for` loops', () => {
-      /*var or let? counter = 100*/
-      /*for (var or let? counter = 1; counter < 50; counter++){}*/
-
-      //expect(counter).toBe(50);
+      var counter = 100 /*var or let? counter = 100*/
```

Comment:
Aca tenias que comentar esta linea, the esta diciendo que var no restringe solo al interior del scope de la funcion y funciona como global